apps/api/src/domain/models/bibliographic-reference/entities/BookBibliographicReferenceData.ts

```diff
@@ -11,11 +22,62 @@ export default class BookBibliographicReferenceData
     readonly type = BibliographicReferenceType.book;
 
     @IsStringWithNonzeroLength()
-    title: string;
+    readonly title: string;
+
+    @ArrayNotEmpty()
+    readonly creators: Creator[];
+
+    @IsOptional()
+    @IsStringWithNonzeroLength()
+    // `abstractNote` is what Zotero calls this property
+    readonly abstract?: string;
+
+    @IsOptional()
+    @IsYear()
+    readonly year?: number;
+
+    @IsOptional()
+    @IsStringWithNonzeroLength()
+    readonly publisher: string;
+
+    @IsOptional()
+    @IsStringWithNonzeroLength()
+    readonly place: string;
+
+    @IsOptional()
+    @IsUrl()
+    readonly url: string;
+
+    @IsOptional()
+    @IsInt()
```

Comment:
We do not want to stack validation decorators (`IsOptional` is behavioural, so it's ok). What we want is a
`IsNonNegativeInteger` decorator. Justin needed the same on his branch. Put a todo here to refactor later.

Reply:
Will leave this to Justin's branch where it was resolved.